securesignals-gma-devapp/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Lets remove this and the associated xml drawables.

Reply:
Done.